scripts/test-upgrade.sh

```diff
@@ -148,6 +149,17 @@ echo "YES" | ${OA_DIR}/scripts/run-upgrade.sh
 popd
 
 # TASK #8
+# Set upgrade variables for the RPCO playbooks
+# The variables are put into a temporary user_variables file, then deleted
+# at the end of this script.
+touch ${UPGRADE_VARIABLES_FILE}
+if grep -q 'logging_upgrade' ${UPGRADE_VARIABLES_FILE}; then
+  sed -i "s/logging_upgrade:.*/logging_upgrade: true/" ${UPGRADE_VARIABLES_FILE}
```

Comment:
I'm not sure if there will be white space before the first letter so I've just added a `$` to prevent multiline matching.